### PR TITLE
feat(frontend): use sendBtc service

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Use sendBtc service if all conditions are met.

### CODE REVIEW

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 1]
```diff
@@ -48,8 +58,75 @@
...
+		}
+
+		if (isNullishOrEmpty(destination)) {
+			toastsError({
+				msg: { text: $i18n.send.assertion.destination_address_invalid }
+			});
+			return;
+		}
+
+		if (invalidAmount(amount) || isNullish(amount)) {
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:78]: from the code `invalidAmount` already checks for nullish

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 2]
```diff
@@ -48,8 +58,75 @@
...
+			toastsError({
+				msg: { text: $i18n.send.error.unexpected }
+			});
+			return;
+		}
+
+		dispatch('icNext');
+
+		try {
+			await sendBtc({
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:109]: just for posterity, may you please add a TODO for tracking analytics?

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
Just two small suggestion/requests, but the rest LGTM!

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:78]: Somehow TS requires to have additional isNullish check. I guess for the same reason it is done like this in IcSendTokenWizard.

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:109]: Sure.

File: src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte [hunk 3]
```diff
@@ -48,8 +58,76 @@
...
+
+		if (isNullish($sendToken)) {
+			toastsError({
+				msg: { text: $i18n.tokens.error.unexpected_undefined }
+			});
+			return;
+		}
+
+		if (isNullish($authIdentity)) {
+			toastsError({
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte:100]: Shouldn't `await nullishSignOut();` be used here?

**Pull request merged**